docs/src/main/asciidoc/azure-functions-http.adoc

```diff
@@ -50,68 +71,92 @@ If you don't log in to Azure you won't be able to deploy.
+In Quarkus dev mode, your application just runs as an HTTP endpoint without integration with Azure Functions.
 
 [source,bash,subs=attributes+]
 ----
 ./mvnw clean package quarkus:dev
 ----
 
+== Run the application locally in an Azure Functions simulated environment
 
-
-== Run locally in Azure Functions simulated environment
-
-If you want to try your app with a simulated local Azure Functions environment, you can
-use this command
+If you want to try your app with a simulated local Azure Functions environment, use the following command:
 
 [source,bash,subs=attributes+]
 ----
 ./mvnw clean package azure-functions:run
 ----
 
-Note that you must have the https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local#v2[Azure Functions Core Tools]
-installed for this to work!
+[IMPORTANT]
+====
+You must have the https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local#v2[Azure Functions Core Tools]
+installed for this to work.
+====
 
-== Deploy to Azure
+== Deploy to Azure Functions
 
-The `pom.xml` you generated in the previous step pulls in the `azure-functions-maven-plugin`.  Running maven package
-generates config files and a staging directory required by the `azure-functions-maven-plugin`.  Here's
-how to execute it.
+The `pom.xml` that you generated in the previous step pulls in the Azure Functions Maven plugin (`azure-functions-maven-plugin`).
+Running the Maven package generates the required configuration files and staging directory.
+
+Use the following Quarkus CLI command to deploy the package to Azure Functions:
 
 [source,bash,subs=attributes+]
 ----
 ./mvnw clean package azure-functions:deploy
 ----
 
-If deployment is a success, the azure plugin will tell you the base URL to access your function.
+== Access your function
+
+When the deployment completes, the Azure Functions Maven plugin returns the base URL to access your function.
+
+.Example
 
-i.e.
 [source]
 ----
 [INFO] HTTP Trigger Urls:
 [INFO] 	 QuarkusHttp : https://{appName}.azurewebsites.net/api/{*path}
 ----
 
-The URL to access the service would be
+In the example provided, the URL to access the service is:
 
-https://{appName}.azurewebsites.net/api/hello
+\https://{appName}.azurewebsites.net/api/hello
 
-== Extension maven dependencies
 
-You must include the `quarkus-azure-functions-http` extension as this is a generic bridge between the Azure Functions
-runtime and the HTTP framework you are writing your microservices in.
+:sectnums!:
 
-== Azure Deployment Descriptors
+== Summary
 
-Templates for Azure Functions deployment descriptors (`host.json`, `function.json`) are within
-the base directory of the project. Edit them as you need to.  Rerun the build when you are ready.
+Congratulations!
+You have learned how to create, build, and deploy a sample Quarkus RESTEasy microservice to Azure Functions.
 
+// Should this step be positioned earlier in this guide? It seems a bit late here...
```

Comment:
If/when we split, this is something for the reference guide, rather than the tutorial..